settings.go

```diff
@@ -2,13 +2,21 @@ package main
 
 import (
 	"errors"
+	"fmt"
 
 	"github.com/kubewarden/gjson"
 	kubewarden "github.com/kubewarden/policy-sdk-go"
-
-	"fmt"
 )
 
+type HostPath struct {
```

Comment:
about what we talked here, we should either document in README.md that `PathPrefix`, `ReadOnly` are optional, and what happens then, or make sure we fail if they are missing.

Reply:
@viccuad at the end I haven't changed the original cide because I realized that this would mean using a mix of `encoding/json` and `gjson`.

This still fails if either  `pathPrefix`  or `ReadOnly` is missing as per the original implementation.

Reply:
I think it's fine to keep the old behavior. We can change it, but with another PR